pom.xml

```diff
@@ -174,9 +174,8 @@
                 <configuration>
                     <imageManagement>
                         <image>
-                            <repository>${dockerHubPublic}/cafapi/opensuse-jre17</repository>
-                            <tag>4.0.1</tag>
-                            <digest>sha256:434c16f975fbbe2f4adfc879d88a32b5f8fcc92a2d8e29e3598dc10088717405</digest>
+                            <repository>${dockerHubPublic}/cafapi/prereleases</repository>
+                            <tag>opensuse-jre21-4.1.0-SNAPSHOT</tag>
```

Comment:
Using the latest snapshot due to an issue found with the permissions on the nssdb causing NSS failure on classification worker. The latest snapshot has the correct permissions.